modules/installation-aws-user-infra-installation.adoc

```diff
@@ -49,7 +49,9 @@ INFO Time elapsed: 1s
 +
 [IMPORTANT]
 ====
-The Ignition config files that the installation program generates contain certificates that expire after 24 hours, which are then renewed at that time. If the cluster is shut down before renewing the certificates and the cluster is later restarted after the 24 hours have elapsed, the cluster automatically recovers the expired certificates. The exception is that you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates. See the documentation for _Recovering from expired control plane certificates_ for more information.
+* The Ignition config files that the installation program generates contain certificates that expire after 24 hours, which are then renewed at that time. If the cluster is shut down before renewing the certificates and the cluster is later restarted after the 24 hours have elapsed, the cluster automatically recovers the expired certificates. The exception is that you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates. See the documentation for _Recovering from expired control plane certificates_ for more information.
+
+* It is recommended that Ignition config files are used shortly after they are generated because certificate rotation can occur at anytime, even before the 24-hour certificate expiration.
```

Comment:
```suggestion
* It is recommended that Ignition config files are used shortly after they are generated to avoid issues where the cluster may attempt to renew certificates during cluster installation. The cluster may renew certificates hours before the 24-hour expiration.
```